src/main/java/core/gui/comp/table/PlayersTable.kt

```diff
@@ -0,0 +1,87 @@
+package core.gui.comp.table
+
+import core.model.HOVerwaltung
+import core.model.player.Player
+import java.beans.PropertyChangeEvent
+import java.beans.PropertyChangeListener
+import java.beans.PropertyChangeSupport
+import javax.swing.event.ListSelectionEvent
+
+class PlayersTable @JvmOverloads constructor(tableModel: HOTableModel, fixedColumnsCount: Int = 1) :
+    FixedColumnsTable(tableModel, fixedColumnsCount), PropertyChangeListener {
+
+        companion object {
+
+            private val propertyChangeSupport : PropertyChangeSupport = PropertyChangeSupport(this)
+
+            fun addPropertyChangeListener(listener: PropertyChangeListener?) {
+                propertyChangeSupport.addPropertyChangeListener(listener)
+            }
+
+            fun removePropertyChangeListener(listener: PropertyChangeListener?) {
+                propertyChangeSupport.removePropertyChangeListener(listener)
+            }
+
+            fun firePropertyChanged(oldSelection: Player?, newSelection: Player?){
+                propertyChangeSupport.firePropertyChange("SelectedPlayer", oldSelection, newSelection)
+            }
+
+            private var selectedPlayer: Player? = null
+
+            fun setSelectedPlayer(player: Player?){
+                if ( player != selectedPlayer){
+                    val oldSelection = selectedPlayer
+                    selectedPlayer = player
+                    firePropertyChanged(oldSelection, player)
+                }
+            }
+
+            fun getSelectedPlayer() : Player?{
+                return selectedPlayer
+            }
+        }
+
+    var players: MutableList<Player?> = HOVerwaltung.instance().getModel().getCurrentPlayers()
```

Comment:
Thinking about this one, this should belong in the tableModel.  Here a generic HOPlayerTableModel, that returns the list of players, as well as tracks the selected players would make sense I think.